android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -187,10 +189,16 @@ public void onClick(View v) {
             Timber.d(date+"\n"+endTime+"\n"+startTime);
 
         }
-        summary.setText(session.getSummary());
+        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.N) {
```

Comment:
Instead of adding this again use one if statement for both summary and description. No need to write same if condition twice.